### DISCUSSION TITLE
[Feature request] Add support for the "grouping" tag

### DISCUSSION THREAD

**Discussion by KingDuckZ** [navidrome/navidrome#2860] [Ideas]
I have many soundtracks with names that don't necessarily refer to the movie/game they relate to. When I tag my music in Picard, I add a "grouping" tag to such albums so that for example both the album "Die Hard OST" and the single "Summer in the city" are related to the same movie.
In other words, I would like something similar to the "genre" filter to browse by "grouping", so that if I want to see all I have on Die Hard I can just filter for that. Ideally that would also be exposed to client players through the API.

**Comment by metalheim** [comment]
This is implemented. Close?

**Reply by deluan** [reply]
Yes. Good catch!
